src/config/configurations.ts

```diff
@@ -5,13 +5,19 @@ import {
   StaticTransportStrategy,
 } from './transport/transport-strategy';
 import {GrpcConfiguration} from './transport/grpc-configuration';
+import {LogFormat, LoggerOptions, LogLevel} from '../utils/logging';
 
 // 4 minutes.  We want to remain comfortably underneath the idle timeout for AWS NLB, which is 350s.
 const defaultMaxIdleMillis = 4 * 60 * 1_000;
 const defaultMaxSessionMemoryMb = 256;
+const defaultLoggerOptions: LoggerOptions = {
+  level: LogLevel.WARN,
+  format: LogFormat.CONSOLE,
+};
 
 export class Laptop extends SimpleCacheConfiguration {
-  static latest() {
+  static latest(loggerOptions: LoggerOptions | undefined = undefined) {
+    const finalLoggerOptions = loggerOptions ?? defaultLoggerOptions;
```

Comment:
I think this is equivalent.

```suggestion
  static latest(loggerOptions: LoggerOptions = defaultLoggerOptions) {
```